DynamoLeagueBlazor/Client/Features/Teams/SignPlayer.razor.cs

```diff
@@ -0,0 +1,77 @@
+﻿using DynamoLeagueBlazor.Shared.Features.Teams;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.WebAssembly.Authentication;
+using Microsoft.AspNetCore.WebUtilities;
+using MudBlazor;
+using System.Net.Http.Json;
+
+namespace DynamoLeagueBlazor.Client.Features.Teams;
+
+public partial class SignPlayer : IDisposable
+{
+    [Inject] private HttpClient HttpClient { get; set; } = null!;
+    [Inject] private ISnackbar SnackBar { get; set; } = null!;
+    [CascadingParameter] MudDialogInstance MudDialogInstance { get; set; } = null!;
+    [Parameter, EditorRequired] public int PlayerId { get; set; }
+    [Parameter, EditorRequired] public EventCallback OnSignPlayerButtonClick { get; set; }
+
+
+
+    private SignPlayerRequest _form = null!;
+    private SignPlayerDetailResult? _signPlayerDetailResult;
+    private bool _processingForm;
+    private readonly CancellationTokenSource _cts = new();
+
+    protected override async Task OnInitializedAsync()
+    {
+        _form = new SignPlayerRequest { PlayerId = PlayerId };
+
```

Comment:
This is misspelled

Reply:
I feel really dumb
don't see what is misspelled